cli/command/cli_test.go

```diff
@@ -170,6 +170,110 @@ func TestExperimentalCLI(t *testing.T) {
 	}
 }
 
+func TestOrchestratorSwitch(t *testing.T) {
+	defaultVersion := "v1.55"
```

Comment:
nit: use `v0.00` to more clearly state it's just a random version?

Reply:
Ok